src/js/fluxcontroller.js

```diff
@@ -374,6 +381,9 @@ define(function (require, exports, module) {
                 }
 
                 return modalPromise
+                    .catch(function () {
+                        // If modal state already ended, quietly continue
```

Comment:
I think this is safe, though I really wish we had some better way of catching the expected errors only instead of all the errors.